BUILD.bazel

```diff
@@ -369,14 +383,14 @@ py_proto_library(
 ################################################################################
 
 alias(
-    name = "conformance_test_runner",
-    actual = "//conformance:conformance_test_runner",
+    name = "test_messages_proto2_proto",
+    actual = "//src/google/protobuf:test_messages_proto2_proto",  # proto_library
     visibility = ["//visibility:public"],
 )
 
 alias(
-    name = "test_messages_proto2_proto",
-    actual = "//src/google/protobuf:test_messages_proto2_proto",  # proto_library
+    name = "test_messages_proto2_proto_srcs",
+    actual = "//src/google/protobuf:test_messages_proto2.proto",
     visibility = ["//visibility:public"],
```

Comment:
I think this new target should probably be private since we don't necessarily want external code to be able to use our internal unit test protos (it's too late for the existing ones but we might as well lock down the new ones).

Now that I think of it, do we even need an alias here? I believe most of the other aliases in this file were added just for compatibility when targets were moved to src/google/protobuf.

Reply:
Is it too late for the existing ones?  This is already a breaking change

Reply:
I'm ok with cleaning some of this up now

Reply:
I removed all of these from the root BUILD.bazel file